### DISCUSSION TITLE
Modifying and deploying a custom version of Spoke

### DISCUSSION THREAD

**Discussion by FarhadG** [mozilla/hubs#2726] [General Questions & Discussion] [answered]
We have a custom hubs cloud instance in AWS and would like to make changes to Spoke. Is there a way we can have a custom Spoke instance running within our instance?

**Comment by robinkwilson** [comment, marked as answer]
Currently, we don't have an easy way to customize Spoke per HC instance. Quickly made an issue in the Spoke repo for you to follow (please thumbs up to vote for it!): https://github.com/mozilla/Spoke/issues/1006

If you make anything for Spoke that might be helpful to others, please consider creating a PR or a feature request in the Spoke repo! We'd really appreciate it!

**Reply by FarhadG** [reply]
This would be wonderful and I'd definitely contribute if making and deploying changes is available. Looking forward to seeing this functionality
